README.md

```diff
@@ -55,6 +55,32 @@ await ws_test.run()
 assert ws_test.is_complete()
 ```
 
+Testing a response with the following more complicated body is received on connection to a websocket host:
```

Comment:
with or when?

Reply:
With, but I've changed the sentence a little to make it a bit more clear